conf/modules/actuators_dummy.xml

```diff
@@ -10,6 +10,7 @@
     </description>
   </doc>
   <dep>
+    <!--depends>actuators</depends-->
```

Comment:
Can this be removed?

Reply:
I'm planing to change the actuators interface, so I'd rather clean this later